### ISSUE TITLE
List SHU token on Optimism bridge

### ISSUE THREAD

**Issue by 0xneves** [blockful-io/shutter-bridge-op#2]
@update
[PR to add SHU Token on OP Superchain](https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/907)

Add a pull request to the [data folder](https://github.com/ethereum-optimism/ethereum-optimism.github.io/tree/master/data) with the information needed and described in the docs.

What's needed before starting:
- Resolve the [task](https://github.com/orgs/blockful-io/projects/9/views/1?pane=issue&itemId=68619754) of getting the token address on L2
- Token icon

Resources:
- [Docs](https://static.optimism.io/)
- [Question on Op forum about this](https://help.optimism.io/hc/en-us/articles/9932349706139-How-do-I-add-my-token-to-the-Superchain-bridge-front-ends)
- [Post on Shutter forum](https://shutternetwork.discourse.group/t/proposal-to-deploy-a-dex-pool-on-a-low-cost-chain-part-3/370)

**Comment by 0xneves** [comment]
Need this data to move this issue forward:
- SHU round token image as PNG and 50x50 size

@Gabrielnvk <3

**Comment by Gabrielnvk** [comment]
@0xneves Shutter DAO new Logo 

![Image](https://github.com/user-attachments/assets/f5b0596a-a66b-4828-bc9f-4534821112c9)

**Comment by 0xneves** [comment]
It needs to be a rounded logo like this:

![Image](https://github.com/user-attachments/assets/e4d4bca9-c87e-42eb-a91c-cea9611af6c2)

Sorry for not specifying. It must be invisible outside the border.
You can also submit svg files.

The resulting logo will be displayed like this:

![Image](https://github.com/user-attachments/assets/bcb4ddf0-7d2e-4b01-8305-c08200345c73)

**Comment by alextnetto** [comment]
![Image](https://github.com/user-attachments/assets/9c7c9ae0-f0df-4b7c-8253-915a762ff50a)

**Comment by 0xneves** [comment]
> ![Image](https://private-user-images.githubusercontent.com/56097505/354333520-9c7c9ae0-f0df-4b7c-8253-915a762ff50a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4OTEzNzYsIm5iZiI6MTcyMjg5MTA3NiwicGF0aCI6Ii81NjA5NzUwNS8zNTQzMzM1MjAtOWM3YzlhZTAtZjBkZi00YjdjLTgyNTMtOTE1YTc2MmZmNTBhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA1VDIwNTExNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiMDliMjRkNjlkYWJlMzI0YjlkOTBjNmIzNzE5YjZhM2ZlMmM5MDBjZWUwZWIyZWZkNGM4OGMyNzEwZDgyYTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.381uY8tKD2XphIiggEu5Ri9pfnYSFD6VmDjaS41uC7w)


That definitely won't do
![Image](https://github.com/user-attachments/assets/5d149c3b-7452-4e01-acaf-78313442cea3)

Example of ShibaDoge as a 350x350 png image:
![Image](https://github.com/user-attachments/assets/2df91a8e-efcb-4ae2-a0f9-5c9026742621)

**Comment by 0xneves** [comment]
https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/907
